core/context/src/main/java/com/milesight/beaveriot/context/integration/model/BaseEntityBuilder.java

```diff
@@ -24,6 +24,12 @@ public class BaseEntityBuilder<T extends BaseEntityBuilder> {
     protected Map<String, Object> attributes;
     protected String parentIdentifier;
 
+    protected Boolean visible = true;
+
+    protected Boolean customized = false;
+
+    protected Long workflowId;
```

Comment:
1. Specifies whether the workflow Id is required
2. customized is for internal use only and does not need to be in the builder ? 
3. visible does not contain the set method?

Reply:
Fixed.